test/data/biosamples_to_sites.json

```diff
@@ -183,37 +183,38 @@
   ],
   "field_research_site_set": [
     {
-      "id": "GOLD:BESC-13-CL1_35_33",
+      "id": "nmdc:frsite-2ec2ebde-857f-4dc5-ab5e-881c3eb337ce",
+      "name": "BESC-13-CL1_35_33",
       "description": "Bioscales tree BESC-13-CL1_35_33"
     },
     {
-      "id": "GOLD:BESC-13-CL2_39_29",
+      "id": "nmdc:frsite-10e5281b-504b-4f56-80fe-2e50d7d00521",
+      "name": "BESC-13-CL2_39_29",
       "description": "Bioscales tree BESC-13-CL2_39_29"
     }
   ],
   "collecting_biosamples_from_site_set": [
     {
-      "id": "cleanroom_schema:cbfs1",
+      "id": "nmdc:clsite-4ff95e3a-e5e1-4a85-96b2-acf3fb242004",
       "name": "Collection of biosamples from BESC-13-CL1_35_33",
       "has_inputs": [
-        "GOLD:BESC-13-CL1_35_33"
+        "nmdc:frsite-2ec2ebde-857f-4dc5-ab5e-881c3eb337ce"
       ],
       "has_outputs": [
-        "GOLD:Gb0305833",
-        "GOLD:Gb0291692",
-        "GOLD:Gb0291582"
+        "nmdc:bsm-ad9d77a0-52d7-43fe-ba66-6fd33b6946d2",
+        "nmdc:bsm-2a85c34d-ef39-4160-8959-ce831b26c1fa",
+        "nmdc:bsm-c253f1e2-d532-4fdb-9bd3-32b611dcb680"
       ]
     },
     {
-      "id": "cleanroom_schema:cbfs2",
+      "id": "nmdc:clsite-87c6ff79-4d89-4362-be87-67766356b1ce",
       "name": "Collection of biosamples from BESC-13-CL2_39_29",
       "has_inputs": [
-        "GOLD:BESC-13-CL2_39_29"
+        "nmdc:frsite-10e5281b-504b-4f56-80fe-2e50d7d00521"
       ],
       "has_outputs": [
-        "GOLD:Gb0305834",
-        "GOLD:Gb0291693",
-        "GOLD:Gb0291583"
+        "nmdc:bsm-f34aa0dd-1fbf-4c46-aa16-713d41cfdc64",
+        "nmdc:bsm-32dea169-9135-4e9b-bce2-2286ac10ac0b"
       ]
```

Comment:
This had 3 GOLD ids and now only has 2 nmdc biosample ids so missing one id.

Reply:
The `has_outputs` of the two `CollectingBiosamplesFromSite` instances account for all six of the Biosamples now.